src/providers/v3/gas-data-provider.ts

```diff
@@ -33,16 +29,15 @@ export type OptimismGasData = {
 };
 
 export class OptimismGasDataProvider
-  implements IL2GasDataProvider<OptimismGasData>
-{
+  implements IL2GasDataProvider<OptimismGasData> {
   protected gasOracleAddress: string;
 
   constructor(
     protected chainId: ChainId,
     protected multicall2Provider: IMulticallProvider,
     gasPriceAddress?: string
   ) {
-    if (chainId != ChainId.OPTIMISM) {
+    if (chainId !== ChainId.OPTIMISM && chainId !== ChainId.BASE) {
       throw new Error('This data provider is used only on optimism networks.');
```

Comment:
on optimism or base network.

Reply:
This is the inverse, not optimism and not base